JumpRoyale/src/ArenaBuilder.cs

```diff
@@ -0,0 +1,76 @@
+using Godot;
+
+public class ArenaBuilder(TileMap tileMap)
+{
+    /// <summary>
+    /// Defines at what Y in tiles the object type should change. For reference, the screen is 63 tiles tall on a 1080p
+    /// screen and Y goes negative upwards. Important note: the current arena is placed one screen below the default
+    /// position, so the actual next screen is located at Y=0. The estimate average max height the players reach
+    /// during the game is around 3500px (actually -155 height in the game), so the last texture change should be
+    /// "almost" at the estimated max height. Despite the screen being 63 tiles tall, we change the object type every 50
+    /// tiles to give players a chance to see all the textures. With 63~ tiles per change, the last change occurs at
+    /// 4000px, which will barely be seen by players.
+    /// </summary>
+    private readonly int[] _objectTypeChangeHeights = [0, -50, -100, -150];
+    private readonly TileMap _tileMap = tileMap;
+
+    public void DrawPlatform(int x, int y, int width)
+    {
+        // Note: the smallest platform, for visual purposes, will always be 2 tiles wide, because we have to draw the
+        // left and the right side. The Width only determines the extension of the platform, which is the middle part
+        // and when we say "Draw Platform of Width 1", we mean a platform that is 1 tile wide + the left and right side.
```

Comment:
This should be a function-level comment (i.e. a doc comment) so that caller will know what `width` represents.

Reply:
Moved this to the summary 👍 please check if that's what your suggestion meant.

Reply:
Yup, that's indeed what I meant. 👍